alevel7/EpicMail
# Ch babel implementation #164661869

#### What does this PR do?
- Integrates babel and allow transpiling. Code syntax changed to es6.Entry point also changed

#### Description of Task to be completed?

- Installed babel7 and dependencies
- changed code syntax to es6 in all javascript files
- readjusted some written test



#### How should this be manually tested?

repository should be cloned and server should be started using npm start
if it should start without error then it works. npm test should also be executed





#### What are the relevant pivotal tracker stories?

Integrate babel with es6 for transpiling-#164661869 (https://www.pivotaltracker.com/n/projects/2314800
